src/utilities.cpp

```diff
@@ -1292,19 +1282,19 @@ bool linkComponentVariableUnits(const ComponentPtr &component, std::vector<Issue
 
 bool traverseComponentEntityTreeLinkingUnits(const ComponentEntityPtr &componentEntity)
 {
-    std::vector<IssuePtr> issueList;
+    DescriptionList issueList;
```

Comment:
`issueList` is a bad variable name.

Reply:
Nothing to do with this PR as such, but ok done in commit b41a9e7a.